Authorization/Authorization.py

```
@@ -0,0 +1,49 @@
+
+def authorization(f):
+    @wraps(f)
+    def authorizeTheUser(*args,**kwargs):
+        try:
+            global result
+            # print(request.cookies.get(os.getenv('ACCESS_TOKEN_KEY')))
+            print("Request Headers: ",request.headers.get("Authorization"))
+            auth_header=request.headers.get('Authorization')
+            if auth_header:
+              #  verify the access token
+                result=requests.get(f"{os.getenv('GET_SET_TOKENS_URL')}/decode-token",json={os.getenv('ACCESS_TOKEN_KEY'):auth_header}).json()
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)